app/scrapers/__init__.py

```diff
@@ -40,7 +42,7 @@ def small_test():
 
 
 def feedgen(query, engine, count=10):
-    if engine in ['q', 't']:
+    if engine in ['q', 't', 's']:
```

Comment:
@raju249 don't include `Stackoverflow` here.
These are done for Quora and YouTube which doesn't have page indexing . But StackOverflow has

Reply:
Sure

Reply:
But the scraper is not allowed to go immediately to second page. Its redirected to captcha page due to which the scraper given 0 urls after first result

Reply:
That's interesting @raju249. Try changing the user agent, it might work.

Reply:
Yes @AnshnulMalik 
I just tried it again and some calls were redirected to captcha, for other i got the results.

Reply:
> But the scraper is not allowed to go immediately to second page. 

It can only be done when you will introduce `startKey` and `defaultKey` in `stackoverflow.py` [Still I am not able to see this change in your branch] and remove this `s`

Reply:
Alright, give me a sec, sending new updated PR

Reply:
How about waiting for few seconds before the next get request then?

Reply:
I think this will work. But that would also delay the results rendering to front end

Reply:
How about using the api then ?

Reply:
Please push same thing to heroku app, if its working locally

Reply:
I added a delay of 1 sec, it worked like charm. @AnshulMalik 
And we can use the API, but that would need to create a different flow just only for StackOverFlow. Since API response will be different as compared to Scraping html page.

Reply:
So, we can't rely on delaying, also it doesn't seem good.

I am a bit busy these days(exams) so won't be able to devote time, I can look into this after mid december, till then you guys can figure out someting.

Reply:
Sure

Reply:
We are redirected to captcha page after first results. Screenshot below. 

![screenshot from 2017-11-29 10-35-59](https://user-images.githubusercontent.com/8307266/33358849-236abc6a-d4f1-11e7-8c6e-4ca38d6621ad.png)